examples/2016-10-31/image_resize_golang/template.yaml

```diff
@@ -0,0 +1,54 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: A function is triggered off an upload to a bucket. It uploads a resized image to another bucket.
+
+Globals:
+  Function:
+    Timeout: 20
+
+Parameters:
+  BucketNamePrefix:
+    Type: String
+    Default: sam-exmaple
+
+Resources:
+  ImageProcessorFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: .
+      Handler: main
+      Runtime: go1.x
+      Tracing: Active
+      Policies:
+      - S3ReadPolicy:
+          BucketName: !Sub "${BucketNamePrefix}-source-bucket"
+      - S3CrudPolicy:
+          BucketName: !Sub "${BucketNamePrefix}-dest-bucket"
+      Environment:
+        Variables:
+          DestBucket: "sam-exmaple-dest-bucket"   # BucketNamePrefix-dest-bucket
```

Comment:
Needs to be `DestBucket: !Sub "${BucketNamePrefix}-dest-bucket"` as well in case a value other than the default is used.

Reply:
Thanks for the advise. I have updated the template file according to the instruction.

Reply:
Thanks for the advice. I have updated the template file according to the instruction.